server/manager/src/test/java/org/apache/accumulo/manager/upgrade/RootFilesUpgradeTest.java

```diff
@@ -78,6 +78,7 @@ public void renameReplacement(VolumeManager fs, Path tmpDatafile, Path newDatafi
       rename(fs, tmpDatafile, newDatafile);
     }
 
+    @SuppressWarnings("removal")
```

Comment:
Instead of suppressing the old property, can the test be updated to use the new?